src/improvements/template/EnableDeputyPauseModuleTemplate.sol

```diff
@@ -20,7 +20,7 @@ contract EnableDeputyPauseModuleTemplate is SimpleBase {
     address public newModule;
 
     /// @notice Constant safe address string identifier
-    string constant _SAFE_ADDRESS = "FoundationOperationSafe";
+    string constant _SAFE_ADDRESS = "FoundationOperationsSafe";
```

Comment:
Since we have only FoS I am wondering yet why Operation**s**. 
This is my first review I will delete this comment If I found why.

Reply:
In the specs we call it "Foundation Operations Safe" - unifying the naming here to account for that.